x-pack/plugins/canvas/__tests__/fixtures/function_specs.ts

```diff
@@ -4,9 +4,13 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+// @ts-ignore Library missing type definition
 import { Fn } from '@kbn/interpreter/common';
 import { functions as browserFns } from '../../canvas_plugin_src/functions/browser';
 import { functions as commonFns } from '../../canvas_plugin_src/functions/common';
 import { functions as serverFns } from '../../canvas_plugin_src/functions/server';
+import { AvailableFunctions } from '../../canvas_plugin_src/functions/types';
 
-export const functionSpecs = [...browserFns, ...commonFns, ...serverFns].map(fn => new Fn(fn()));
+export const functionSpecs: AvailableFunctions[] = [...browserFns, ...commonFns, ...serverFns].map<
+  AvailableFunctions
+>(fn => new Fn(fn()));
```

Comment:
What's the best way to handle getting things typed appropriately when working with an untyped function like `Fn` in this case?

Should we `as` the result to what we expect it to be? Or leave it being any and letting the `.map` type do it's thing and let it break once `Fn` becomes properly typed?